src/Lecture2.hs

```diff
@@ -97,7 +116,8 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+dropSpaces :: String -> String
+dropSpaces = (takeWhile $ not.isSpace).(dropWhile isSpace)
```

Comment:
I would add spaces around `.` to be consistent with common Haskell formatting standards and to future-proof against changes in the Haskell compiler. Also, brackets can be rearranged:

```suggestion
dropSpaces = takeWhile (not . isSpace) . dropWhile isSpace
```